Celeste.Mod.mm/Mod/Everest/Everest.Updater.cs

```diff
@@ -277,6 +314,7 @@ public class Source {
                     return;
                 }
 
+                CoreModule.Settings.CurrentBranch = version.Source.Name;
```

Comment:
This setting is not actually written to the file
perhaps move it until after the installation succeeds and call `CoreModule.Instance.SaveSettings()` manually